ocp_resources/resource.py

```diff
@@ -453,6 +453,17 @@ def to_dict(self):
 
     def __enter__(self):
         signal(SIGINT, self._sigint_handler)
```

Comment:
Should signal handler be reverted upon __exit__?

Reply:
No, we register it at start.